src/main/java/stringCalculator/Tokenizer.java

```diff
@@ -0,0 +1,27 @@
+package stringCalculator;
+
+public class Tokenizer {
+
+    public static String[] splitToken;
+
+    public static void setSplitToken(String token){
+        splitToken = token.split(" ");
+    }
+
+    public static Operator[] separateToken() {
+        Operator[] operators = new Operator[splitToken.length / 2];
+
+        for (int i = 1; i < splitToken.length; i += 2) {
+            operators[i/2] = createOperator(splitToken[i]);
+        }
+
+        return operators;
+    }
+
+    public static Operator createOperator(String symbol){
+        if(Operator.operatorCollection.containsKey(symbol)) {
+            return new Operator(symbol);
+        }
+        throw new NotVaildOperatorException("올바른 사칙연산 기호를 입력해주세요");
```

Comment:
```suggestion
        if(!Operator.operatorCollection.containsKey(symbol)) {
            throw new NotVaildOperatorException("올바른 사칙연산 기호를 입력해주세요");
        }
        return new Operator(symbol);
```
이렇게 처리하는게 더 자연스러울 것 같습니다.